cairo-vm-cli/src/main.rs

```diff
@@ -41,7 +41,9 @@ struct Args {
 
 fn validate_layout(value: &str) -> Result<(), String> {
     match value {
-        "plain" | "small" | "dex" | "bitwise" | "perpetual_with_bitwise" | "all" => Ok(()),
+        "plain" | "small" | "dex" | "bitwise" | "perpetual_with_bitwise" | "all" | "dynamic" => {
```

Comment:
The layouts changed with my PR, this should change also.

Reply:
Done!